src/dlls/clretwrc/CMakeLists.txt

```diff
@@ -12,8 +12,7 @@ endif(WIN32)
 
 add_definitions(-DFX_VER_INTERNALNAME_STR=clretwrc.dll)
 
-add_library(clretwrc SHARED
-  clretwrc.rc
+add_library_clr(clretwrc SHARED
```

Comment:
This is missing the clretwrc.rc file, that's why the build in the lab is failing.